src/iidm/VoltageLevel.cpp

```diff
@@ -245,8 +260,10 @@ void VoltageLevel::remove() {
     // Remove the topology
     removeTopology();
 
-    // Remove this voltage level from the network
-    getSubstation().remove(*this);
+    if (getSubstation()) {
```

Comment:
It's not necessary to call the getter here? It's useful in Java to map the return of the getter and use the fluent API of the `Optional` class

Reply:
Agree. Fixed